src/components/Input.scss

```diff
@@ -158,5 +159,39 @@ $form-input-bg-color: $cu-foreground--dark !default;
       @include ss-pika;
     }
   }
+}
 
+// * 5. Errors and feedback
+.Input-feedback {
+  display: none;
+}
+.is-required.is-invalid {
+  .Input-label:after {
+    color: $cu-negative;
+  }
 }
+.is-required.is-valid {
+  .Input-label:after {
+    content: ' ';
+  }
+}
+.Input.is-invalid {
+  .Input-element {
+    border-color: $cu-negative;
+  }
+  .Input-feedback {
+    display: block;
+    color: $cu-negative;
+  }
+}
+.Input.is-valid {
+  svg {
```

Comment:
Can we add a class to this instead of referencing the element directly? I could see a scenario where there might be other SVG within an Input component.